.github/workflows/coverage.yml

```diff
@@ -119,7 +119,10 @@ jobs:
 
       - name: Generate code coverage
         run: |
-          QW_ENV=LOCAL cargo +nightly llvm-cov --all-features --workspace --lcov --output-path lcov.info
+          cargo +nightly llvm-cov clean --workspace
```

Comment:
Should work according the docs https://github.com/taiki-e/cargo-llvm-cov

Reply:
I triggered manually the coverage here: https://github.com/quickwit-inc/quickwit/actions/runs/1655778045

Reply:
this run is still failling